test/export/test_export.py

```diff
@@ -1200,7 +1200,7 @@ def forward(self, x, y):
                 return max(x.shape[0], y.shape[0])
 
         model = Foo()
-        inputs = (torch.randn(64), torch.randn(64))
+        inputs = (torch.zeros(64), torch.ones(64))
```

Comment:
why?

Reply:
Likely not needed, this was to make the test deterministic. In very rare cases, they could result in same answer causing the runtime assertion to not fired from max() call.